<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLabel" name="checksumStatus">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General telemetry</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>GPS longitude</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_longitude"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>GPS latitude</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_latitude"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>GPS altitude</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_altitude"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Pressure</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_pressure"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>BMP180 temperature</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_bmp180_temperature"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>External temperature</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_external_temperature"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Voltage</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lineEdit_voltage"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Radiation</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="lineEdit_radiation"/>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Internal temperature</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="lineEdit_internal_temperature"/>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DS18B20 temperature 1</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="lineEdit_ds18b20_temperature1"/>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DS18B20 temperature 2</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="lineEdit_ds18b20_temperature2"/>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DS18B20 temperature 3</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QLineEdit" name="lineEdit_ds18b20_temperature3"/>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DS18B20 temperature 4</string>
              </property>
             </widget>
            </item>
            <item row="12" column="1">
             <widget class="QLineEdit" name="lineEdit_ds18b20_temperature4"/>
            </item>
            <item row="13" column="0">
             <widget class="QLabel" name="label_15">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Acceleration</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="label_14">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Angular speed</string>
              </property>
             </widget>
            </item>
            <item row="13" column="1">
             <widget class="QWidget" name="widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLineEdit" name="lineEdit_accelerometer_x"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_accelerometer_y"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_accelerometer_z"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="14" column="1">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLineEdit" name="lineEdit_gyroscope_x"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_gyroscope_y"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_gyroscope_z"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <item>
             <property name="text">
              <string>STM32</string>
             </property>
             <property name="background">
              <brush brushstyle="NoBrush">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GPS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Habduino DS18B20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SDIO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SD Card</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FatFS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EEPROM</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Humidity sensor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>I2C</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SMBus</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Accelerometer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Magnetometer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gyroscope</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IR temperature sensor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sun sensor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Geiger counter</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dust sensor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Camera small</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Camera big</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>IMU</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QOpenGLWidget" name="openGLWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Lock map position</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Controls</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>90</y>
          <width>381</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>Activate parachute</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>140</y>
          <width>381</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>Explode</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
